apps/web-giddh/src/assets/css/style-2.scss

```diff
@@ -1409,6 +1423,10 @@ modal-container.modal {
     padding-top: 85px !important;
 }
 
+.mb-0 {
```

Comment:
class d-none is already predefined in bootstrap.css @prateek-giddh

Reply:
Done